CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ And please only add new entries to the top of this list, right below the `# Unre
 - On Wayland, a new `wayland-csd-adwaita-crossfont` feature was added to use `crossfont` instead of `ab_glyph` for decorations.
 - On Wayland, if not otherwise specified use upstream automatic CSD theme selection.
 - On Windows, fixed ALT+Space shortcut to open window menu.
+- On X11, added `WindowExtUnix::with_x11_parent` to create child windows.
```

Comment:
Nit:
```suggestion
- On X11, added `WindowExtX11::with_x11_parent` to create child windows.
```

On that note, maybe rename to just `with_parent`?

Reply:
Thanks! Fixed and Renamed it.